packages/icons/stories/icon.doc.stories.mdx

```diff
@@ -34,14 +34,14 @@ If an icon is interactive, use [Button](?path=/docs/documentation-core-button--b
 
 ## Features
 
-We have a default icon set that starts from 12px for HD & MD, then 14px for LD, 16px for TD. (Since our base size of HD/default is 10px, but we don’t allowing icon size that is smaller then 12px, hence why here, the HD/default will start from 12px, and its 2X is 20px, 3X is 30px etc.)
-Have more size choices by adding multiplies to 2x, 3x, 4x etc.
+We have a default icon set that starts from 12px for high- and medium density (MD), then 14px for low density (LD), and 16px for touch density (TD). (Since our base size of high density (HD)/default is 10px but we don’t allow icons size smaller than 12px, hence why here the HD/default will start from 12px, and its 2X is 20px, 3X is 30px etc.)
```

Comment:
```suggestion
We have a default icon set that starts from 12px for high and medium density (MD), then 14px for low density (LD), and 16px for touch density (TD). (Since our base size of high density (HD)/default is 10px but we don’t allow icons size smaller than 12px, hence why here the HD/default will start from 12px, and its 2X is 20px, 3X is 30px etc.)
```